docs/src/pages/guides/migration-v4/migration-v4.md

```diff
@@ -168,6 +168,23 @@ const theme = createMuitheme({
 });
 ```
 
+- Breakpoints are now treated as values instead of ranges. The behavior of `down(key)` was changed to define media query less than the value defined with the corresponding breakpoint (exclusive). Tbe `between(start, end)` was also updated to define media query for the values between the actual values of start (inclusive) and end (exclusive). Find examples of the changes required defined below:
```

Comment:
```suggestion
- Breakpoints are now treated as values instead of ranges. The behavior of `down(key)` was changed to define media query less than the value defined with the corresponding breakpoint (exclusive).
  The `between(start, end)` was also updated to define media query for the values between the actual values of start (inclusive) and end (exclusive).
  Find examples of the changes required defined below:
```